Modules/Loadable/SlicerWelcome/qSlicerWelcomeModuleWidget.cxx

```diff
@@ -233,6 +232,12 @@ void qSlicerWelcomeModuleWidget::setup()
     {
     d->OpenExtensionsManagerButton->hide();
     }
+#else
+  d->OpenExtensionsManagerButton->hide();
+#endif
+
+  this->Superclass::setup();
```

Comment:
Accidental double `this->Superclass::setup();`?

Reply:
Yes, good catch @jamesobutler 👍